packages/mimir/test/basic_test.dart

```diff
@@ -380,13 +380,13 @@ void main() {
 
   test('numberOfDocuments returns the correct number of documents', () async {
     final index = await useTestIndex();
-    expect(await index.numberOfDocuments, 0);
+    expect(await index.numberOfDocuments, BigInt.from(0));
```

Comment:
I'm guessing FRB was updated to now return `BigInt` instead?

Reply:
Yes